packages/allegro5/allegro5.0.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "OCaml Allegro 5"
+description: "An OCaml binding of the Allegro 5 game & multimedia C library"
+maintainer: ["Sylvain Chiron <chironsylvain@orange.fr>"]
+authors: ["Sylvain Chiron"]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/Frigory33/ocaml-allegro-5"
+bug-reports: "https://github.com/Frigory33/ocaml-allegro-5/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.11"}
+  "odoc" {with-doc}
+  "conf-allegro5"
```

Comment:
To address the test failure in https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/ec7ebf3d7e33040e1e6f69592341b57233ab0173/variant/distributions,alpine-3.20-ocaml-4.14,allegro5.0.1,tests which looks like

```
https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/ec7ebf3d7e33040e1e6f69592341b57233ab0173/variant/distributions,alpine-3.20-ocaml-4.14,allegro5.0.1,tests
```

I think we'll need to declare the `dune-site` package as a test dependency. This should do the trick:

```suggestion
  "conf-allegro5"
  "dune-site" {with-test}
```

Reply:
For future releases, you can reflect this change back into your https://github.com/Frigory33/ocaml-allegro-5/blob/main/dune-project